2024-10-04 00:11:32,447 - CRITICAL - [in /home/erpfarm/workspace/farm-iot/data_processor.py:84] 
 Dữ liệu process_data: {'mac_address': 'GF_TN4_2_CAMERA3', 'action_name': 'end', 'timer': '86'}
 Mã trạng thái HTTP server: 200, Phản hồi từ server: 200
2024-10-04 00:11:32,450 - CRITICAL - [in /home/erpfarm/workspace/farm-iot/data_processor.py:96] Đã cập nhật dữ liệu about: 9838eee342a8
2024-10-04 07:12:05,261 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv3 -> start
2024-10-04 07:42:46,095 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv3 -> end
2024-10-04 08:38:58,008 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> start
2024-10-04 09:35:54,754 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> end
2024-10-04 09:35:55,562 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> start
2024-10-04 10:10:37,237 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> end
2024-10-04 11:01:39,213 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 14:36:48,013 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 15:04:59,265 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 15:16:42,578 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> start
2024-10-04 16:37:06,717 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 16:37:25,297 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 16:38:04,147 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 16:40:57,653 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:41:06,218 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 16:41:06,443 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:42:59,332 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 16:44:52,169 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> start
2024-10-04 16:44:58,703 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 16:45:10,378 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:45:18,348 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 16:45:20,537 - ERROR - [in /home/erpfarm/workspace/farm-iot/app.py:357]  
 Quá nhiều yêu cầu từ thiết bị: 181134ab4c24_uv2
2024-10-04 16:45:29,636 - ERROR - [in /home/erpfarm/workspace/farm-iot/app.py:357]  
 Quá nhiều yêu cầu từ thiết bị: 181134ab4c24_uv2
2024-10-04 16:45:30,385 - ERROR - [in /home/erpfarm/workspace/farm-iot/app.py:357]  
 Quá nhiều yêu cầu từ thiết bị: 181134ab4c24_uv2
2024-10-04 16:45:40,214 - ERROR - [in /home/erpfarm/workspace/farm-iot/app.py:357]  
 Quá nhiều yêu cầu từ thiết bị: 181134ab4c24_uv2
2024-10-04 16:46:00,105 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:46:01,730 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 16:46:17,539 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> start
2024-10-04 16:46:19,160 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 16:46:22,206 - ERROR - [in /home/erpfarm/workspace/farm-iot/app.py:357]  
 Quá nhiều yêu cầu từ thiết bị: 181134ab4c24_uv2
2024-10-04 16:46:23,015 - ERROR - [in /home/erpfarm/workspace/farm-iot/app.py:357]  
 Quá nhiều yêu cầu từ thiết bị: 181134ab4c24_uv2
2024-10-04 16:47:23,293 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> start
2024-10-04 16:47:23,341 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:47:23,536 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv3 -> end
2024-10-04 16:47:23,635 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv4 -> end
2024-10-04 16:47:37,234 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> start
2024-10-04 16:47:40,183 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:48:22,600 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> end
2024-10-04 16:48:22,801 - ERROR - [in /home/erpfarm/workspace/farm-iot/app.py:357]  
 Quá nhiều yêu cầu từ thiết bị: 181134ab4c24_uv2
2024-10-04 16:49:02,009 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:49:23,370 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv1 -> end
2024-10-04 16:49:23,435 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv2 -> end
2024-10-04 16:49:23,616 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv3 -> end
2024-10-04 16:49:23,713 - ERROR - [in /home/erpfarm/workspace/farm-iot/data_processor.py:106]  
 Không tìm thấy thông tin cho ID: 181134ab4c24 -> uv4 -> end
2024-10-04 17:48:54,799 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 17:49:19,819 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 17:50:58,020 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 17:51:20,824 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 17:52:08,048 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
2024-10-04 17:52:38,390 - CRITICAL - [in /home/erpfarm/workspace/farm-iot/data_processor.py:84] 
 Dữ liệu process_data: {'mac_address': 'GF_TN4_2_CAMERA1', 'action_name': 'start', 'timer': '86'}
 Mã trạng thái HTTP server: 200, Phản hồi từ server: 200
2024-10-04 17:52:38,391 - CRITICAL - [in /home/erpfarm/workspace/farm-iot/data_processor.py:96] Đã cập nhật dữ liệu about: 9838eee342a8
2024-10-04 19:23:27,488 - CRITICAL - [in /home/erpfarm/workspace/farm-iot/data_processor.py:84] 
 Dữ liệu process_data: {'mac_address': 'GF_TN4_2_CAMERA1', 'action_name': 'end', 'timer': '86'}
 Mã trạng thái HTTP server: 200, Phản hồi từ server: 200
2024-10-04 19:23:27,489 - CRITICAL - [in /home/erpfarm/workspace/farm-iot/data_processor.py:96] Đã cập nhật dữ liệu about: 9838eee342a8
2024-10-04 22:24:07,998 - INFO - [in /home/erpfarm/workspace/farm-iot/app.py:46] Start: GF-CICO
